CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 _Fixes:_
 
 - Made chart userOptions available within `customCode` as variable `options` [(#551)](https://github.com/highcharts/node-export-server/issues/551).
+- Fix the base64 images not working in exported SVGs (Namespace prefix xlink for href on image is not defined, [#547](https://github.com/highcharts/node-export-server/issues/547)).
```

Comment:
Before merging, this should be moved to the `# 4.0.2` version, since there was a small hot fix and `v4.0.2` is already released.

Reply:
I think now that we've added 4.0.2 (on master), it is as expected.